2024Project-main/2024FinalYearProject/Controllers/ClientController.cs

```diff
@@ -126,10 +151,21 @@ public async Task<IActionResult> CashSent(CashSentViewModel model)
             // Save the transaction
             await _repo.Transaction.AddAsync(transaction);
 
+            // Create a notification for Cash Sent
+            var notification = new Notification
+            {
+                Message = "You have successfully sent cash. Amount: {model.Amount:C}.",
```

Comment:
I think you are missing a "$" before this string or you left it this way intentionally?

Reply:
i will sovle that and commit to this branch